boards/arduino-leonardo/src/lib.rs

```diff
@@ -172,20 +177,23 @@ pub mod pwm {
     pub use atmega32u4_hal::pwm::*;
 }
 
+pub type Baudrate = hal::usart::Baudrate<hal::clock::MHz16>;
+
```

Comment:
Please drop this type alias, it is not needed due to the `*Ext` traits you added.

```suggestion
```